src/main/java/tasks/TaskList.java

```diff
@@ -0,0 +1,46 @@
+package tasks;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a list of tasks. It has methods for handling the tasks,
+ * such as adding a task, removing a task, getting a task,
+ * returning the list of tasks, and setting a task to be done.
+ */
+
+public class TaskList {
+    protected ArrayList<Task> tasks;
```

Comment:
Consider using List<Task> instead, in case you want to use a different implementation of List in the future.